example/app.js

```diff
@@ -5,19 +5,24 @@ import { render } from 'react-dom';
 import { Admin, Resource } from 'admin-on-rest';
 import jsonRestClient from 'aor-json-rest-client';
 import { Delete } from 'admin-on-rest/mui';
+import { resolveBrowserLocale } from 'admin-on-rest';
 
 import { PostList, PostCreate, PostEdit, PostShow, PostIcon } from './posts';
 import { CommentList, CommentEdit, CommentCreate, CommentIcon } from './comments';
 
 import data from './data';
+import * as messages from './i18n';
 
 const restClient = jsonRestClient(data, true);
-const delayedRestClient = (type, resource, params) => new Promise(resolve => setTimeout(() => resolve(restClient(type, resource, params)), 1000));
+
+const delayedRestClient = (type, resource, params) => new Promise(
+    resolve => setTimeout(() => resolve(restClient(type, resource, params)), 1000),
+);
 
 render(
-    <Admin restClient={delayedRestClient} title="Example Admin">
-        <Resource name="posts" list={PostList} create={PostCreate} edit={PostEdit} show={PostShow} remove={Delete} icon={PostIcon} />
-        <Resource name="comments" list={CommentList} create={CommentCreate} edit={CommentEdit} remove={Delete} icon={CommentIcon} />
+    <Admin restClient={delayedRestClient} title="Example Admin" locale={resolveBrowserLocale()} messages={messages}>
```

Comment:
What happens if I don't pass a `locale`? Is the default locale used, and the default (English) messages displayed?